vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -108,6 +108,7 @@ provision_actions:
     - "ci_test_provision"
     - "provision_cluster"
     - "provision_controlplane"
+    - "provision_worker"
```

Comment:
why do we need it here?

Reply:
Otherwise the templates don't get updated when we provision workers. It's an omission from a previous change.

Reply:
https://github.com/metal3-io/metal3-dev-env/pull/785/files missed one of the provisioning actions for when we should generate templates.